doc/manual/source/physics/active_particles.rst

```diff
@@ -0,0 +1,137 @@
+.. _active_particles:
+
+
+Active Particles
+============================================
+
+Active Particles (APs) were introduced into Enzo in version 2.6. They were originally part of the
+of the Enzo-3.0 effort and were developed by a number of authors including Nathan Goldbaum,
+Matt Turk, John Wise, John Regan, Oliver Hahn and others. Their design philosophy was that the
```

Comment:
please add Brian Crosby and Greg Meece to this list!  They contributed quite a few bug fixes back to Nathan.

Reply:
Good call. I'll update.